Code/GraphMol/RGroupDecomposition/Wrap/rdRGroupComposition.cpp

```diff
@@ -288,7 +288,9 @@ struct rgroupdecomp_wrapper {
         "input structure\n"
         "    - doEnumeration: expand input cores into enumerated mol bundles\n"
         "    - allowMultipleRGroupsOnUnlabelled: permit more than one rgroup to "
-        "be attached to an unlabelled core atom";
+        "be attached to an unlabelled core atom\n"
+        "    - allowMultipleCoresInSameMol: permit a core to match more than "
```

Comment:
Perhaps: permit multiple cores to match in the molecule if the set of atoms matched are non-equal.